chain/store/store.go

```diff
@@ -21,6 +21,12 @@ import (
 	"github.com/nknorg/nkn/v2/util/log"
 )
 
+const (
+	beforeFastSync uint32 = iota
```

Comment:
Don't use iota for persist state as it can change when you change code order. Use explicitly value